File: XamarinMazeSolvingApp/XamarinMazeSolvingApp/Turn.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace XamarinMazeSolvingApp
{
    class Turn
    {
        public Direction direction;
        public TimeSpan time;

        public Turn(Direction direction, int millis)
        {
            this.direction = direction;
            time = TimeSpan.FromMilliseconds(millis);
        }
    }
}

File: AndroidMazeSolverApp/Mazesolver/app/src/main/java/eu/janrebe/mazesolver/Turn.java
package eu.janrebe.mazesolver;

public class Turn {
    public Direction direction;
    public int timeMillis;

    public Turn(Direction direction, int timeMillis)
    {
        this.direction = direction;
        this.timeMillis = timeMillis;
    }

    @Override
    public String toString() {
        return "Turn: direction: " + direction.toString() + ", time: " + timeMillis;
    }
}
